src/app/shared/blockly/blockly-editor/blockly-editor.component.ts

```diff
@@ -125,11 +130,20 @@ export class BlocklyEditorComponent implements AfterViewInit {
       setTimeout(() => this.blocklyOnLoad(), 500);
     } else {
       this.workspace = (window as any).Blockly.init(this.blocklyId, this.toolbox);
-      this.loadingElement.nativeElement.remove();
-      this.initBlockly();
+      setTimeout(() => {
+        this.onResize();
+        this.loadingElement.nativeElement.remove();
```

Comment:
This is not very good solution. I recommend you to create property
```public loading$ = new BehaviorSuject(true);```
and set it there to false;

And in the component:
```
<div ...... *ngIf="loading$ | async">
.....
```